interface/forms/prior_auth/FormPriorAuth.class.php

```diff
@@ -10,7 +10,9 @@
  * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
  */
 
-use OpenEMR\Common\ORDataObject\ORDataObject;
+ use OpenEMR\Common\ORDataObject\ORDataObject;
+
+ require_once($srcdir . "/formatting.inc.php");
```

Comment:
remove this since always included :)
https://github.com/openemr/openemr/blob/master/composer.json#L91